src/FilterMonitor.js

```diff
@@ -10,10 +10,11 @@ export default class FilterMonitor extends Component {
     blacklist: PropTypes.array
   };
 
-  isFiltered(action) {
+  isFiltered(actionType) {
+    var type = actionType || '';
```

Comment:
If there's no way to prevent that, I'll merge it. Just it's strange that Redux DevTools don't throw.